Gopkg.toml

```diff
@@ -20,10 +20,11 @@ required = [
 
 [[constraint]]
   name = "sigs.k8s.io/cluster-api"
+  #revision = "0734939e05aeb64ab198e3feeee8b4e90ee5cbb2"
   branch = "master"
```

Comment:
I think it better to use tag `0.0.0-alpha.4` because it includes all recent changes and will prevent broken API on `deps update`

Reply:
That release does not contain https://github.com/kubernetes-sigs/cluster-api/pull/598. It's been unwritten rule to take a specific revision or master HEAD til now. Given we usually need the latest changes in the project, taking anything than master HEAD can be problematic.

Reply:
I see your point but can we at least stick to some commit on the master?

Reply:
If we do it here, we need to do it everywhere else. Though, we can wait until the next release is out and update all deps at once in all repos.

Reply:
Otherwise, the exact sha of the cluster-api is in the Godep.lock file.